submodules/LIB/hardware/syn/Makefile

```diff
@@ -42,7 +49,10 @@ UFLAGS+=COV=$(COV)
 UFLAGS+=COV_TEST=$(COV_TEST)
 
 build: $(VHDR) $(VSRC) config.tcl
-ifeq ($(SYN_SERVER),)
+ifeq ($(SYNTHESIZER),yosys)
+	make pdk_install
```

Comment:
so everytime you run synth you need to install the pdk? doesn't make much sense...

Reply:
if you cant use nix we need to find a better solution

Reply:
Volare it's the open source pdk package manager, it's installed in nix. Maybe I can download the pdk to be inside the nix files. Install pdk with volare it basicly checks if the pdk is locally installed, if it's not it will download the pdk, if it's installed it will do nothing.